<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindowUiRoot</class>
 <widget class="QMainWindow" name="MainWindowUiRoot">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Project Starter</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout" stretch="3,7">
      <property name="spacing">
       <number>1</number>
      </property>
      <item>
       <widget class="ProNameList" name="project_name_list" native="true"/>
      </item>
      <item>
       <widget class="RelpaceView" name="replace_view" native="true"/>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="BrowseTextbox" name="out_put_text_box" native="true"/>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <widget class="ProgressBar" name="progress" native="true"/>
        </item>
        <item>
         <widget class="QDialogButtonBox" name="buttonBox">
          <property name="standardButtons">
           <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menusetting">
    <property name="title">
     <string>設定</string>
    </property>
    <addaction name="action_setting"/>
   </widget>
   <addaction name="menusetting"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="action_setting">
   <property name="text">
    <string>setting</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>BrowseTextbox</class>
   <extends>QWidget</extends>
   <header>browse_textbox.hpp</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>ProNameList</class>
   <extends>QWidget</extends>
   <header>pro_name_list.hpp</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>RelpaceView</class>
   <extends>QWidget</extends>
   <header>relpace_view.hpp</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>ProgressBar</class>
   <extends>QWidget</extends>
   <header location="global">progress_bar.hpp</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
